diffrax/brownian/tree.py

```diff
@@ -88,69 +144,173 @@ def __init__(
             )
         self.key = split_by_tree(key, self.shape)
 
+    def _denormalise_bm_inc(self, x: LevyVal) -> LevyVal:
+        # TODO: demonstrate rescaling actually helps
+
+        # Rescaling back from [0, 1] to the original interval [t0, t1].
+        interval_len = self.t1 - self.t0  # can be any dtype
+        sqrt_len = jnp.sqrt(interval_len)
+
+        def sqrt_mult(z):
+            # need to cast to dtype of each leaf in PyTree
+            return (z * sqrt_len).astype(z.dtype)
+
+        def mult(z):
+            return z * (interval_len).astype(z.dtype)
+
+        return LevyVal(
+            h=jtu.tree_map(mult, x.h),
+            W=jtu.tree_map(sqrt_mult, x.W),
+            J=jtu.tree_map(mult, x.J),
+            H=jtu.tree_map(sqrt_mult, x.H),
+        )
+
     @eqx.filter_jit
     def evaluate(
-        self, t0: Scalar, t1: Optional[Scalar] = None, left: bool = True
-    ) -> PyTree[Array]:
-        del left
+        self,
+        t0: Scalar,
+        t1: Optional[Scalar] = None,
+        left: bool = True,
+        use_levy: bool = False,
+    ) -> LevyVal:
+        """Computes the Brownian increment between times `t0` and `t1`.
+
+        **Arguments:**
+            - `t0`: Start of interval
+
+            - `t1`: End of interval
+
+            - `left`: ignored since Brownian motion is continuous
+
+            - `use_levy`: If true, then the return type is a LevyVal encapsulating
+            the Brownian increment and the corresponding Lévy area if it was computed.
```

Comment:
I don't think you need a docstring here; it should inherit it from its parent class.

Reply:
Fair.